### PR TITLE
DEV-8303 Added assertions for end to end tests. Update the readme fil…

### PR DESCRIPTION
**Pull request by MikeMcGarry**
…e. Add Dockerfile and Travis.yml file for testing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,5 @@
 *.csv
```
Comment (rizwansaeed) [inline .gitignore:1]: is it work prefixing with `**/` to exclude from all folders?

File: performance_engine/config/config.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"fields" : {
+		"ClientInc" : "Portfolio/JLH/ClientInc",
```
Comment (rizwansaeed) [inline performance_engine/config/config.json:3]: is `JLH` intrinsically tied into the code?

File: .gitignore (hunk 1 above)
Comment (MikeMcGarry) [reply .gitignore:1]: Tested this, that is not necessary, it already excludes them deep into the folder structure.

File: performance_engine/config/config.json (hunk 2 above)
Comment (MikeMcGarry) [reply performance_engine/config/config.json:3]: Nope.

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**